src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/AccessRequirementRelatedProjectsList.java

```diff
@@ -15,22 +16,26 @@ public class AccessRequirementRelatedProjectsList implements IsWidget {
 
   ReactComponentDiv container;
   public IsACTMemberAsyncHandler isACTMemberAsyncHandler;
+  public SynapseReactClientFullContextPropsProvider propsProvider;
 
   @Inject
   public AccessRequirementRelatedProjectsList(
-    IsACTMemberAsyncHandler isACTMemberAsyncHandler
+    IsACTMemberAsyncHandler isACTMemberAsyncHandler,
+    SynapseReactClientFullContextPropsProvider propsProvider
   ) {
     container = new ReactComponentDiv();
     this.isACTMemberAsyncHandler = isACTMemberAsyncHandler;
+    this.propsProvider = propsProvider;
     container.setVisible(false);
   }
 
   public void configure(String accessRequirementId) {
     AccessRequirementRelatedProjectsListProps props =
       AccessRequirementRelatedProjectsListProps.create(accessRequirementId);
-    ReactNode component = React.createElementWithThemeContext(
+    ReactNode component = React.createElementWithSynapseContext(
```

Comment:
Bug 2:  Pass the full Synapse Context, including the access token.  Without this, only Public projects were able to be resolved in the related projects component.

Reply:
Thanks for catching this!